packages/dd-trace/src/proxy.js

```diff
@@ -55,8 +56,11 @@ class Tracer extends NoopProxy {
 
     try {
       const config = new Config(options) // TODO: support dynamic code config
+
       telemetry.start(config, this._pluginManager)
 
+      if (config.debug) checkRequireCache()
```

Comment:
It only matters that we're _loaded_ before them, not that we _init_ before them, right? So we should maybe just do this at the top-level.

cc @rochdev

Reply:
This has been moved to right before we start patching things